README.md

```diff
@@ -34,6 +34,21 @@ A Home Assistant integration with the Personio API.
 * Install "Personio" as an integration in your settings.
 
 
+## Setup
+
+To use the integration, you need to have API access enabled on your Personio account. You might need to reach out to your People/HR department to request that.
+
+1. Access your settings panel and click on "Generate new credential"
+2. Specify an application identifier. You'll need that later.
+3. You need to enable Read and Write Access to the "Attendances" data. Nothing more is needed.
```

Comment:
This is not true.
You need to have employee READ permission to figure out UID based on email.